evalml/tests/component_tests/test_lsa.py

```diff
@@ -152,20 +152,6 @@ def test_int_col_names():
     assert X_t.dtypes.all() == np.float64
 
 
-def test_repeat_col_names():
```

Comment:
Was removed because Woodwork disallows duplicate column names, so no need to test if this still works--Woodwork will throw an error instead (https://github.com/FeatureLabs/woodwork/issues/377)